src/textParser/tabListener.go

```diff
@@ -0,0 +1,136 @@
+package textParser
+
+import (
+	"fmt"
+	"github.com/antlr4-go/antlr/v4"
+	"strings"
+	"tgScoreBot/src/textParser/parser"
+)
+
+var _ parser.TabListener = (*tabListener)(nil)
+
+type TabProcessor interface {
+	// Bpm sets bpm
+	Bpm(bpm string)
+
+	// Tuning sets guitar tuning
+	Tuning(tuning []string)
+
+	// Duration sets current note duration
+	Duration(duration []string)
+
+	// PlayFret processes tab entry
+	PlayFret(tab string)
+
+	// PlayChord processes a chord
+	PlayChord(chord []string)
+
+	// Pause processes pause
+	Pause()
+
+	// ModelBend models string bending
+	ModelBend(tab string, bendValue string)
+
+	// Errors returns accumulated errors
+	Errors() []error
+
+	// Finish marks the end of parsing
+	Finish()
+}
+
+type tabListener struct {
+	*parser.BaseTabListener
+	processors []*TabProcessor
```

Comment:
Указатель на интерфейс очень странная конструкция, не советую
```suggestion
	processors []TabProcessor
```